### PR TITLE
Miscellaneous fixes

### PR DESCRIPTION
**Pull request by olafurpg**
See the individual commits. Nothing huge here, just my progress for today that is merge-ready.

## Test plan
n/a

### CODE REVIEW

File: plugins/cody-chat/src/com/sourcegraph/cody/chat/agent/CodyManager.java [hunk 1]
```diff
@@ -64,15 +67,6 @@ public void withAgent(Consumer<CodyAgent> action) {
...
   }
 
-  public void onConfigChange(ExtensionConfiguration config) {
-    this.config = config;
-    CodyLogger.onConfigChange(config);
-    if (agentHolder.get() != null) {
-      // Notify agent about config change
-      withAgent(OnFailure.LOG, agent -> agent.server.extensionConfiguration_didChange(config));
-    }
-  }
```
Comment (jamesmcnamara) [inline plugins/cody-chat/src/com/sourcegraph/cody/chat/agent/CodyManager.java:74]: Why is this removed? The only issue I see is that the CodyLogger.onConfigChange sets the endpoint in the log which is a nice-to-have.

Comment (jamesmcnamara) [reply plugins/cody-chat/src/com/sourcegraph/cody/chat/agent/CodyManager.java:74]: Oh I see. Somewhere else we stopped calling this method entirely, so we're never setting the endpoint.

File: plugins/cody-chat/src/com/sourcegraph/cody/chat/agent/CodyAgentClientImpl.java [hunk 2]
```diff
@@ -138,7 +140,9 @@ public CompletableFuture<Void> secrets_delete(Secrets_DeleteParams params) {
   }
 
   @Override
-  public void debug_message(DebugMessage params) {}
+  public void debug_message(DebugMessage params) {
+    log.info(String.format("%s: %s", params.channel, params.message));
```
Comment (jamesmcnamara) [inline plugins/cody-chat/src/com/sourcegraph/cody/chat/agent/CodyAgentClientImpl.java:144]: This does lead to every debug message showing up in the Cody Log twice (once as info and once as a sent message)

image

**Review by jamesmcnamara — COMMENTED** [review]
@olafurpg I added one commit to this to set the endpoint in the CodyLogger after initialization. Feel free to revert it if you see something wrong with that.

**Review by jamesmcnamara — APPROVED** [review]

File: plugins/cody-chat/src/com/sourcegraph/cody/chat/agent/CodyManager.java (hunk 1 above)
Comment (olafurpg) [reply plugins/cody-chat/src/com/sourcegraph/cody/chat/agent/CodyManager.java:74]: The endpoint is set from the webview now. Previously, we set it from the native Eclipse UI to manage account endpoint/access_token pairs.

File: plugins/cody-chat/src/com/sourcegraph/cody/chat/agent/CodyAgentClientImpl.java (hunk 2 above)
Comment (olafurpg) [reply plugins/cody-chat/src/com/sourcegraph/cody/chat/agent/CodyAgentClientImpl.java:144]: We should remove the webview ones along with the filters in the logs, these are not helpful, we should use the JSON-RPC trace.json instead to inspect raw traffic

**Comment by olafurpg** [comment]
LGTM, I missed that the logger needed the endpoint

File: plugins/cody-chat/src/com/sourcegraph/cody/chat/agent/CodyAgentClientImpl.java (hunk 2 above)
Comment (olafurpg) [reply plugins/cody-chat/src/com/sourcegraph/cody/chat/agent/CodyAgentClientImpl.java:144]: Can you take a look at this? [CODY-3559: Remove webview protocol logs from "Cody Logs"](https://linear.app/sourcegraph/issue/CODY-3559/remove-webview-protocol-logs-from-cody-logs)

Comment (jamesmcnamara) [reply plugins/cody-chat/src/com/sourcegraph/cody/chat/agent/CodyAgentClientImpl.java:144]: Sure!
